doc/releases/changelog-dev.md

```diff
@@ -155,6 +155,10 @@
 
 <h3>Internal changes ⚙️</h3>
 
+* The `grad` and `jacobian` primitives now store the function under `fn`. There is also now a single `grad_p`
+  primitive for use in program capture.
+  [(#8357)](https://github.com/PennyLaneAI/pennylane/pull/8357)
```

Comment:
Wouldn't it make more sense to have the single primitive be called `jacobian_p`? 
As far as I know the naming, "gradient" implies scalar outputs of the original function.